ngraph/test/backend/gelu.in.cpp

```diff
@@ -30,62 +30,84 @@
 #include "util/test_control.hpp"
 #include "util/test_tools.hpp"
 
+#include "util/engine/test_engines.hpp"
```

Comment:
Please remove duplication of includes above. Also remove includes related to backend, runtime tensor and util_all_close* which are not needed anymore